html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label><br />
+        <input type="email" /><br />
+      </div>
+
+      <div>
+        <label for="radio-button">Why is coding cool</label><br />
+        <select id="is-coding-cool" name="simple">
+          <br />
+          <option selected>Select an option</option>
+          <option>Becuase it&#39;s fun</option>
+          <option>Because it&#39;s interesting</option>
+          <option>Because it&#39;s powerful</option></select
+        ><br /><br />
+      </div>
+
+      <div>
+        <label for="fav-lang">Favorite Language &#40;choose one&#41;</label
+        ><br /><br />
+        <input type="radio" name="language" id="html" />
+        <label for="html">HTML</label>
+        <input type="radio" name="language" id="css" />
+        <label for="css">CSS</label>
+        <input type="radio" name="language" id="javascript" />
+        <label for="javascript">JavaScript</label><br /><br />
+      </div>
+      <div>
+        <label for="signup"
+          >Would you like to sign up for more information?</label
+        ><br /><br />
+        <input type="checkbox" />
+        <label for="signup">Sign me up!</label>
```

Comment:
You have two `label`s directing to the same `input` (the checkbox), but there's nothing on the `input` that connects the `label` to it. You need to add an attribute to `input` for this to work.